core/include/mm/tee_mmu_types.h

```diff
@@ -111,7 +113,7 @@ struct vm_info {
 
 static inline void mattr_perm_to_str(char *str, size_t size, uint32_t attr)
 {
-	if (size < 7)
+	if (size < 8)
```

Comment:
With this change `user_mode_ctx_print_mappings()` needs to be updated.

Reply:
Sorry, I forgot to remove this change. I was using it during testing to check the attributes. I removed the change in user_mode_ctx_print_mappings() but forgot this one here. Do you think it would be valuable to print the G bit here ? If yes, I will add the change in user_mode_ctx_print_mappings()